qualang_tools/config/waveform_tools.py

```diff
@@ -0,0 +1,63 @@
+import numpy as np
+
+# Definition of pulses follow Chen et al. PRL, 116, 020501 (2016)
+
+
+def drag_gaussian_pulse_waveforms(
+    amplitude, length, sigma, alpha, detune, delta, substracted
+):
+    """
+    Function that returns DRAG pulses that compensate for leakage and also for the AC stark shift
+    that the leakage creates to the qubit frequency.
+
+    Envelope: gaussian
+
+    :param amplitude: amplitude of voltage
```

Comment:
add types for all parameters:
:param double amplitude"
:param int length"
etc

Reply:
will do